front-end/src/util/sorting.js

```diff
@@ -1,19 +1,41 @@
+export function sortByLastMetDate(leftCraftsperson, rightCraftsperson) {
```

Comment:
i think everywhere it is always referred to as lastMeeting, I would rename all of these to that

Reply:
done